src/containers/shared/utils.js

```diff
@@ -78,6 +78,54 @@ const FORMAT_PRICE_DEFAULT_OPTIONS = {
   padding: 0,
 }
 
+export const isLaterVersion = (source, target) => {
+  if (source === ' N/A ') return false
+  if (target === ' N/A ') return true
+  const sourceDecomp = source.split('.')
+  const targetDecomp = target.split('.')
+  // Compare major version
+  if (parseInt(sourceDecomp[0], 10) > parseInt(targetDecomp[0], 10)) return true
+  // Compare minor version
+  if (
+    parseInt(sourceDecomp[0], 10) === parseInt(targetDecomp[0], 10) &&
+    parseInt(sourceDecomp[1], 10) > parseInt(targetDecomp[1], 10)
+  )
+    return true
+  const sourcePatch = sourceDecomp[2].split('-')
+  const targetPatch = targetDecomp[2].split('-')
+  // Compare patch version
+  if (
+    parseInt(sourceDecomp[0], 10) === parseInt(targetDecomp[0], 10) &&
+    parseInt(sourceDecomp[1], 10) === parseInt(targetDecomp[1], 10)
```

Comment:
This logic can be simplified in two main ways:
1. Since you're returning booleans, you can return conditionals instead of returning `true` and `false` directly. 
2. You can rely on your return values in order to avoid checking for the same conditions twice.

Ex. 
Psuedocode:
1. Are the major versions the same? If NOT -> Return left > right (Since they aren't the same, this should give you the same output as currently)
2. Are the minor versions the same? If NOT -> Return minorLeft > minorRight
3. Are the patch version the same? If NOT -> Return patchLeft > patchRight

For the `rcb` - you may want to just rely on alphanumeric comparison, since if they release a release candidate `c`, that should be later anyway.

(This was a lot of words, I may have misunderstood some complexity in the problem - if so sorry!)

Reply:
Nice fix! Thanks Jackson! For the `rcb`, I cannot rely solely on alphanumeric sort since 1.10.0-b10 will comes before 1.10.0-b9 even though it should be the other way. I fixed part of the code accordingly to your suggestion.